### ISSUE TITLE
NOTICE of Deprecation unless a good developer wishes to undertake this project

### ISSUE THREAD

**Issue by a-r-db** [tgwizard/sls#597]
TL;DR
- At this time I won't be able to maintain this project.
- [Recommended Alternative pScrobbler](https://github.com/kawaiiDango/pScrobbler)

Full Story
- I am currently working full time as a student.
- If somebody has the time and skills to maintain the project and is interested message me or leave a message here.
- I would gladly pass that information along to the original developer.
- I was not a professional developer when initially undertaking this project, so I realize there are changes that should not have been made so quickly in the project.
- There is still potentially an unknown bug that causes the database to be destroyed upon update via google play. I thought I fixed this issue, but I am still worried about providing an update.
- I even attempted to begin a new project Andscrobbler to amend some of my mistakes with managing and attempting to grow the compatibility and features of simple-last-fm-scrobbler. I began that project at Android 1 and wanted to rebuild the entire app from scratch for maximum compatibility and usability.
- I simply do not have the time.
- Now that I have more understanding of software development. Ideally this project would have been a simple maintenance job, although in order to meet everyone's demands I grew the code too fast and too far outside its original capabilities.
- To the original developer and anyone who contributed to and/or loved this project, I am very sorry to have failed this app.
- To the users I recommend pScrobbler as a reasonable and open source alternative.
